src/display/editor/freetext.js

```diff
@@ -311,13 +320,17 @@ class FreeTextEditor extends AnnotationEditor {
    * onkeydown callback.
    * @param {MouseEvent} event
    */
-  keyup(event) {
+  keydown(event) {
     if (event.key === "Enter") {
       this.enableEditMode();
       this.editorDiv.focus();
     }
   }
 
+  keyup(event) {
+    FreeTextEditor._keyboardManager.exec(this, event);
+  }
```

Comment:
Why do we to use both a `keydown` *and* a `keyup` event listener, since that's not immediately clear to me?